### ISSUE TITLE
[!] Your project's Gradle version is incompatible with the Java version that Flutter is using for Gradle.

### ISSUE THREAD

**Issue by 09u2h4n** [flet-dev/flet#2296]
**Description**

When i use "flet build apk" for build apk it gaves an error "[!] Your project's Gradle version is incompatible with the Java version that Flutter is using for Gradle."
I tried to update gradle.

**Code example to reproduce the issue:**

```python
flet build apk -vv
```

**Describe the results you received:**

```python
Flutter bootstrap directory: /tmp/flet_flutter_build_btDwRGLdZY
Creating Flutter bootstrap project...OK
Customizing app icons and splash images...Copying /home/oguz/python_projects/flet/app/assets/icon.png to /tmp/flet_flutter_build_btDwRGLdZY/images
OK
Generating app icons...Resolving dependencies in /tmp/flet_flutter_build_btDwRGLdZY... (5.2s)
Got dependencies in /tmp/flet_flutter_build_btDwRGLdZY.
Building package executable... (6.9s)
Built flutter_launcher_icons:flutter_launcher_icons.
  ════════════════════════════════════════════
     FLUTTER LAUNCHER ICONS (v0.13.1)
  ════════════════════════════════════════════

• Creating default icons Android
• Overwriting the default Android launcher icon with a new icon
• Overwriting default iOS launcher icon with new icon
Creating Icons for Web...              done
Creating Icons for Windows...          done
Creating Icons for MacOS...            done

✓ Successfully generated launcher icons
OK
Generating splash screens...Building package executable... (8.1s)
Built flutter_native_splash:create.
[Android] Creating default splash images
[Android] Creating dark mode splash images
[Android] Creating default android12splash images
[Android] Creating dark mode android12splash images
[Android] Updating launch background(s) with splash image path...
[Android]  - android/app/src/main/res/drawable/launch_background.xml
[Android]  - android/app/src/main/res/drawable-night/launch_background.xml
[Android]  - android/app/src/main/res/drawable-v21/launch_background.xml
[Android]  - android/app/src/main/res/drawable-night-v21/launch_background.xml
[Android] Updating styles...
[Android]  - android/app/src/main/res/values-v31/styles.xml
[Android] No android/app/src/main/res/values-v31/styles.xml found in your Android project
[Android] Creating android/app/src/main/res/values-v31/styles.xml and adding it to your Android project
[Android]  - android/app/src/main/res/values-night-v31/styles.xml
[Android] No android/app/src/main/res/values-night-v31/styles.xml found in your Android project
[Android] Creating android/app/src/main/res/values-night-v31/styles.xml and adding it to your Android project
[Android]  - android/app/src/main/res/values/styles.xml
[Android]  - android/app/src/main/res/values-night/styles.xml
[iOS] Creating  images
[iOS] Creating dark mode  images
[iOS] Updating ios/Runner/Info.plist for status bar hidden/visible
[Web] Creating images
[Web] Creating images
[Web] Creating background images
[Web] Creating CSS
[Web] Updating index.html
╔════════════════════════════════════════════════════════════════════════════╗
║                       NEED A GREAT FLUTTER DEVELOPER?                      ║
╠════════════════════════════════════════════════════════════════════════════╣
║                                                                            ║
║   I am available!  Find me at https://www.linkedin.com/in/hansonjon/       ║
║                                                                            ║
╚════════════════════════════════════════════════════════════════════════════╝

✅ Native splash complete.
Now go finish building something awesome! 💪 You rock! 🤘🤩
Like the package? Please give it a 👍 here: https://pub.dev/packages/flutter_native_splash

OK
Packaging Python app...Building package executable... (4.6s)
Built serious_python:main.
Running package command
Creating asset directory: /tmp/flet_flutter_build_btDwRGLdZY/app
Copying Python app from /home/oguz/python_projects/flet/app to /tmp/serious_python_tempLBEDUG
Installing dependencies [flet-embed] with pip command to /tmp/serious_python_tempLBEDUG/__pypackages__
Extracting Python distributive from /tmp/cpython-3.11.6+20231002-x86_64-unknown-linux-gnu-install_only.tar.gz to /tmp/hostpython3.11_ZKPXLJ
Compiling Python sources at /tmp/serious_python_tempLBEDUG
Delete unnecessary files with extensions: [.py, .c, .h, .typed, .exe, .so, .a, .pdb, .pyd, .dll]
Delete unnecessary files and directories: [__pycache__, bin]
Creating app archive at /tmp/flet_flutter_build_btDwRGLdZY/app/app.zip from /tmp/serious_python_tempLBEDUG
Deleting temp directory /tmp/serious_python_tempLBEDUG
Deleting Python directory /tmp/hostpython3.11_ZKPXLJ
OK
Building .apk for Android...['/snap/bin/flutter', 'build', 'apk']
Resolving dependencies... (1.0s)
  fl_chart 0.65.0 (0.66.0 available)
  flutter_lints 2.0.3 (3.0.1 available)
  flutter_native_splash 2.3.7 (2.3.9 available)
  lints 2.1.1 (3.0.0 available)
  matcher 0.12.16 (0.12.16+1 available)
  material_color_utilities 0.5.0 (0.8.0 available)
  meta 1.10.0 (1.11.0 available)
  package_info_plus 4.2.0 (5.0.1 available)
  path 1.8.3 (1.9.0 available)
  petitparser 5.4.0 (6.0.2 available)
  test_api 0.6.1 (0.7.0 available)
  toml 0.14.0 (0.15.0 available)
  uuid 4.2.2 (4.3.1 available)
  web 0.3.0 (0.4.0 available)
  web_socket_channel 2.4.0 (2.4.1 available)
  xml 6.3.0 (6.5.0 available)
Got dependencies!
16 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.


FAILURE: Build failed with an exception.

* What went wrong:
Could not open cp_settings generic class cache for settings file '/tmp/flet_flutter_build_btDwRGLdZY/android/settings.gradle' (/home/oguz/.gradle/caches/7.5/scripts/5bahl4yp5mw7lcn2efkltdtgf).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 65

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
Running Gradle task 'assembleRelease'...                         1,789ms

┌─ Flutter Fix ─────────────────────────────────────────────────────────────────────────────────────────────┐
│ [!] Your project's Gradle version is incompatible with the Java version that Flutter is using for Gradle. │
│                                                                                                           │
│ To fix this issue, consult the migration guide at docs.flutter.dev/go/android-java-gradle-error.          │
└───────────────────────────────────────────────────────────────────────────────────────────────────────────┘
Gradle task assembleRelease failed with exit code 1
Deleting Flutter bootstrap directory /tmp/flet_flutter_build_btDwRGLdZY
Error building Flet app - see the log of failed command above.
```

**Describe the results you expected:**

Build an apk file

**Additional information you deem important:**

**Java version (`java --version`)**

```
openjdk 21.0.1 2023-10-17
OpenJDK Runtime Environment (build 21.0.1+12-Ubuntu-222.04)
OpenJDK 64-Bit Server VM (build 21.0.1+12-Ubuntu-222.04, mixed mode, sharing)
```

**Gradle version (`gradle --version`)**

```
------------------------------------------------------------
Gradle 8.5
------------------------------------------------------------

Build time:   2023-11-29 14:08:57 UTC
Revision:     28aca86a7180baa17117e0e5ba01d8ea9feca598

Kotlin:       1.9.20
Groovy:       3.0.17
Ant:          Apache Ant(TM) version 1.10.13 compiled on January 4 2023
JVM:          21.0.1 (Private Build 21.0.1+12-Ubuntu-222.04)
OS:           Linux 5.15.133.1-microsoft-standard-WSL2 amd64
```

**Flutter version (`flutter --version`)**

```
Flutter 3.16.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 78666c8dc5 (12 days ago) • 2023-12-19 16:14:14 -0800
Engine • revision 3f3e560236
Tools • Dart 3.2.3 • DevTools 2.28.4
```

**Flutter doctor (`flutter doctor -v`)**

```
[✓] Flutter (Channel stable, 3.16.5, on Ubuntu 22.04.3 LTS 5.15.133.1-microsoft-standard-WSL2, locale C.UTF-8)
    • Flutter version 3.16.5 on channel stable at /home/oguz/snap/flutter/common/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 78666c8dc5 (12 days ago), 2023-12-19 16:14:14 -0800
    • Engine revision 3f3e560236
    • Dart version 3.2.3
    • DevTools version 2.28.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /home/oguz/android-sdk
    • Platform android-33, build-tools 30.0.3
    • Java binary at: /usr/bin/java
    • Java version OpenJDK Runtime Environment (build 21.0.1+12-Ubuntu-222.04)
    • All Android licenses accepted.

[✗] Chrome - develop for the web (Cannot find Chrome executable at google-chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/linux#android-setup for detailed instructions).

[✓] Connected device (1 available)
    • Linux (desktop) • linux • linux-x64 • Ubuntu 22.04.3 LTS 5.15.133.1-microsoft-standard-WSL2

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 2 categories.
```

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.18.0
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page:
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: /home/oguz/python_projects/flet/venv/lib/python3.10/site-packages
Requires: cookiecutter, flet-runtime, packaging, qrcode, watchdog, websocket-client, websockets
Required-by:
```

**Operating system**:

Linux (WSL)
Ubuntu 22.04.3 LTS on Windows 10 x86_64

**Comment by 09u2h4n** [comment]
Maybe the problem is gradle init?

**Comment by 09u2h4n** [comment]
Okay i think, i might resolve maybe. I think Flet's gradle version is 7.5 so i reinstall java (21 to 18) and gradle (8.5 to 7.5). I retyped "flet build apk -vv" and didn't give an error for now and this is still going on

**Comment by 09u2h4n** [comment]
After that it gaves an error but built an apk file and it works nice. However while building the apk file it gave an error which is 
```
Flutter bootstrap directory: /tmp/flet_flutter_build_rhRiKVFRAj
Creating Flutter bootstrap project...OK
Customizing app icons and splash images...Copying /home/oguz/python_projects/flet/my_app/assets/icon.png to /tmp/flet_flutter_build_rhRiKVFRAj/images
OK
Generating app icons...Resolving dependencies in /tmp/flet_flutter_build_rhRiKVFRAj... (3.6s)
Got dependencies in /tmp/flet_flutter_build_rhRiKVFRAj.
Building package executable... (5.3s)
Built flutter_launcher_icons:flutter_launcher_icons.
  ════════════════════════════════════════════
     FLUTTER LAUNCHER ICONS (v0.13.1)
  ════════════════════════════════════════════

• Creating default icons Android
• Overwriting the default Android launcher icon with a new icon
• Overwriting default iOS launcher icon with new icon
Creating Icons for Web...              done
Creating Icons for Windows...          done
Creating Icons for MacOS...            done

✓ Successfully generated launcher icons
OK
Generating splash screens...Building package executable... (4.6s)
Built flutter_native_splash:create.
[Android] Creating default splash images
[Android] Creating dark mode splash images
[Android] Creating default android12splash images
[Android] Creating dark mode android12splash images
[Android] Updating launch background(s) with splash image path...
[Android]  - android/app/src/main/res/drawable/launch_background.xml
[Android]  - android/app/src/main/res/drawable-night/launch_background.xml
[Android]  - android/app/src/main/res/drawable-v21/launch_background.xml
[Android]  - android/app/src/main/res/drawable-night-v21/launch_background.xml
[Android] Updating styles...
[Android]  - android/app/src/main/res/values-v31/styles.xml
[Android] No android/app/src/main/res/values-v31/styles.xml found in your Android project
[Android] Creating android/app/src/main/res/values-v31/styles.xml and adding it to your Android project
[Android]  - android/app/src/main/res/values-night-v31/styles.xml
[Android] No android/app/src/main/res/values-night-v31/styles.xml found in your Android project
[Android] Creating android/app/src/main/res/values-night-v31/styles.xml and adding it to your Android project
[Android]  - android/app/src/main/res/values/styles.xml
[Android]  - android/app/src/main/res/values-night/styles.xml
[iOS] Creating  images
[iOS] Creating dark mode  images
[iOS] Updating ios/Runner/Info.plist for status bar hidden/visible
[Web] Creating images
[Web] Creating images
[Web] Creating background images
[Web] Creating CSS
[Web] Updating index.html
╔════════════════════════════════════════════════════════════════════════════╗
║                       NEED A GREAT FLUTTER DEVELOPER?                      ║
╠════════════════════════════════════════════════════════════════════════════╣
║                                                                            ║
║   I am available!  Find me at https://www.linkedin.com/in/hansonjon/       ║
║                                                                            ║
╚════════════════════════════════════════════════════════════════════════════╝

✅ Native splash complete.
Now go finish building something awesome! 💪 You rock! 🤘🤩
Like the package? Please give it a 👍 here: https://pub.dev/packages/flutter_native_splash

OK
Packaging Python app...Building package executable... (2.6s)
Built serious_python:main.
Running package command
Creating asset directory: /tmp/flet_flutter_build_rhRiKVFRAj/app
Copying Python app from /home/oguz/python_projects/flet/my_app to /tmp/serious_python_tempIMJWSX
Installing dependencies [flet-embed] with pip command to /tmp/serious_python_tempIMJWSX/__pypackages__
Extracting Python distributive from /tmp/cpython-3.11.6+20231002-x86_64-unknown-linux-gnu-install_only.tar.gz to /tmp/hostpython3.11_OANZLK
Compiling Python sources at /tmp/serious_python_tempIMJWSX
Delete unnecessary files with extensions: [.py, .c, .h, .typed, .exe, .so, .a, .pdb, .pyd, .dll]
Delete unnecessary files and directories: [__pycache__, bin]
Creating app archive at /tmp/flet_flutter_build_rhRiKVFRAj/app/app.zip from /tmp/serious_python_tempIMJWSX
Deleting temp directory /tmp/serious_python_tempIMJWSX
Deleting Python directory /tmp/hostpython3.11_OANZLK
OK
Building .apk for Android...['/snap/bin/flutter', 'build', 'apk']
Resolving dependencies...
  fl_chart 0.65.0 (0.66.0 available)
  flutter_lints 2.0.3 (3.0.1 available)
  flutter_native_splash 2.3.7 (2.3.9 available)
  lints 2.1.1 (3.0.0 available)
  matcher 0.12.16 (0.12.16+1 available)
  material_color_utilities 0.5.0 (0.8.0 available)
  meta 1.10.0 (1.11.0 available)
  package_info_plus 4.2.0 (5.0.1 available)
  path 1.8.3 (1.9.0 available)
  petitparser 5.4.0 (6.0.2 available)
  test_api 0.6.1 (0.7.0 available)
  toml 0.14.0 (0.15.0 available)
  uuid 4.2.2 (4.3.1 available)
  web 0.3.0 (0.4.0 available)
  web_socket_channel 2.4.0 (2.4.1 available)
  xml 6.3.0 (6.5.0 available)
Got dependencies!
16 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.

Checking the license for package NDK (Side by side) 23.1.7779620 in /home/oguz/android-sdk/licenses
License for package NDK (Side by side) 23.1.7779620 accepted.
Preparing "Install NDK (Side by side) 23.1.7779620 (revision: 23.1.7779620)".
"Install NDK (Side by side) 23.1.7779620 (revision: 23.1.7779620)" ready.
Installing NDK (Side by side) 23.1.7779620 in /home/oguz/android-sdk/ndk/23.1.7779620
"Install NDK (Side by side) 23.1.7779620 (revision: 23.1.7779620)" complete.
"Install NDK (Side by side) 23.1.7779620 (revision: 23.1.7779620)" finished.
Checking the license for package CMake 3.18.1 in /home/oguz/android-sdk/licenses
License for package CMake 3.18.1 accepted.
Preparing "Install CMake 3.18.1 (revision: 3.18.1)".
"Install CMake 3.18.1 (revision: 3.18.1)" ready.
Installing CMake 3.18.1 in /home/oguz/android-sdk/cmake/3.18.1
"Install CMake 3.18.1 (revision: 3.18.1)" complete.
"Install CMake 3.18.1 (revision: 3.18.1)" finished.
Checking the license for package Android SDK Platform 31 in /home/oguz/android-sdk/licenses
License for package Android SDK Platform 31 accepted.
Preparing "Install Android SDK Platform 31 (revision: 1)".
"Install Android SDK Platform 31 (revision: 1)" ready.
Installing Android SDK Platform 31 in /home/oguz/android-sdk/platforms/android-31
"Install Android SDK Platform 31 (revision: 1)" complete.
"Install Android SDK Platform 31 (revision: 1)" finished.
/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/serious_python_android/intermediates/merged_native_libs/release/out/lib/armeabi-v7a/libpythonbundle.so': The file was not recognized as a valid object file

/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/serious_python_android/intermediates/merged_native_libs/release/out/lib/x86_64/libpythonbundle.so': The file was not recognized as a valid object file

/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/serious_python_android/intermediates/merged_native_libs/release/out/lib/arm64-v8a/libpythonbundle.so': The file was not recognized as a valid object file

Font asset "CupertinoIcons.ttf" was tree-shaken, reducing it from 283452 to 912 bytes (99.7% reduction). Tree-shaking can be disabled by providing the --no-tree-shake-icons flag when building your app.
Font asset "MaterialIcons-Regular.otf" was tree-shaken, reducing it from 1645184 to 1261080 bytes (23.3% reduction). Tree-shaking can be disabled by providing the --no-tree-shake-icons flag when building your app.
/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/app/intermediates/merged_native_libs/release/out/lib/arm64-v8a/libpythonbundle.so': The file was not recognized as a valid object file

/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/app/intermediates/merged_native_libs/release/out/lib/armeabi-v7a/libpythonbundle.so': The file was not recognized as a valid object file

/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/app/intermediates/merged_native_libs/release/out/lib/x86_64/libpythonbundle.so': The file was not recognized as a valid object file

Caught exception: Already watching path: /home/oguz/snap/flutter/common/flutter/packages/flutter_tools/gradle
e: /home/oguz/.gradle/caches/transforms-3/c4988ff80110f75080a936fc6f044848/transformed/core-1.10.1/jars/classes.jar!/META-INF/core_release.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/oguz/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.8.22/636bf8b320e7627482771bbac9ed7246773c02bd/kotlin-stdlib-1.8.22.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/oguz/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.8.22/636bf8b320e7627482771bbac9ed7246773c02bd/kotlin-stdlib-1.8.22.jar!/META-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/oguz/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.8.22/636bf8b320e7627482771bbac9ed7246773c02bd/kotlin-stdlib-1.8.22.jar!/META-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/oguz/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.8.22/1a8e3601703ae14bb58757ea6b2d8e8e5935a586/kotlin-stdlib-common-1.8.22.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
Running Gradle task 'assembleRelease'...                          602.6s
✓  Built build/app/outputs/flutter-apk/app-release.apk (41.0MB).
OK
Copying build to build/apk directory...OK
Deleting Flutter bootstrap directory /tmp/flet_flutter_build_rhRiKVFRAj
Success!
```
**Separate written version of the errors**
```
/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/serious_python_android/intermediates/merged_native_libs/release/out/lib/armeabi-v7a/libpythonbundle.so': The file was not recognized as a valid object file

/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/serious_python_android/intermediates/merged_native_libs/release/out/lib/x86_64/libpythonbundle.so': The file was not recognized as a valid object file

/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/serious_python_android/intermediates/merged_native_libs/release/out/lib/arm64-v8a/libpythonbundle.so': The file was not recognized as a valid object file

/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/app/intermediates/merged_native_libs/release/out/lib/arm64-v8a/libpythonbundle.so': The file was not recognized as a valid object file

/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/app/intermediates/merged_native_libs/release/out/lib/armeabi-v7a/libpythonbundle.so': The file was not recognized as a valid object file

/home/oguz/android-sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip: error: '/tmp/flet_flutter_build_rhRiKVFRAj/build/app/intermediates/merged_native_libs/release/out/lib/x86_64/libpythonbundle.so': The file was not recognized as a valid object file

Caught exception: Already watching path: /home/oguz/snap/flutter/common/flutter/packages/flutter_tools/gradle
e: /home/oguz/.gradle/caches/transforms-3/c4988ff80110f75080a936fc6f044848/transformed/core-1.10.1/jars/classes.jar!/META-INF/core_release.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/oguz/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.8.22/636bf8b320e7627482771bbac9ed7246773c02bd/kotlin-stdlib-1.8.22.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/oguz/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.8.22/636bf8b320e7627482771bbac9ed7246773c02bd/kotlin-stdlib-1.8.22.jar!/META-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/oguz/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.8.22/636bf8b320e7627482771bbac9ed7246773c02bd/kotlin-stdlib-1.8.22.jar!/META-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/oguz/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.8.22/1a8e3601703ae14bb58757ea6b2d8e8e5935a586/kotlin-stdlib-common-1.8.22.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
```

**Comment by kalatawei** [comment]
**Hello** 
**it works for me by following these steps** : 
1- make a new **windows10 pro VM. (English version)**
2- install **Android Studio 2022.3** (Giraffe)
3- install **Git 2.4** or later.
4- **Flutter SDK 3.16.5** , add path to windows **system environment variables**
5- flet **0.18.0**

**Comment by FeodorFitsner** [comment]
Thanks for the update!

**Comment by FeodorFitsner** [comment]
It is fixed in Flet 0.23.1.

Please cleanup Cookiecutter cache before running `flet build` command.

On Windows:

```
rd /s /q %USERPROFILE%\.cookiecutters
```

On Mac/Linux:

```
rm -rf ~/.cookiecutters
```
